magento/magento2
# Improve bundle load speeds

### Description
When a bundle has a lot of selection, the tax GetPriceConfigurationObserver becomes a real performance hog because it loads the selections over and over again. This fixes that by caching the selection in memory

### Manual testing scenarios
Create a bundle with a lot of selections. Check the loading speed of the product page with the bundle before and after this commit. In my local testing scenarios, the loading time went from ~30s to about ~3s
